pkg/diagnostics/report.go

```diff
@@ -40,4 +55,28 @@ func proxyStatus() {
 			fmt.Printf("	❌ %-12s : not set or empty.\n", proxyName)
 		}
 	}
+	fmt.Println("")
+}
+
+// overridedEnvVar retrieves and displays overridden environment variables with the prefix `CONFIG_JS_` from the noobaa-core-0 pod
+func overridenEnvVar(coreApp *appsv1.StatefulSet, endpointApp *appsv1.Deployment) {
+	log := util.Logger()
+
+	log.Print("⏳ Retrieving overridden environment variable details...\n")
+
+	fmt.Print("Overriden Environment Variables Check (NOOBAA-CORE):\n----------------------------------\n")
+	for _, envVar := range coreApp.Spec.Template.Spec.Containers[0].Env {
+		if strings.HasPrefix(envVar.Name, "CONFIG_JS_") {
+			fmt.Printf("    	✔ %-20s : %s\n", envVar.Name, envVar.Value)
+		}
+	}
+	fmt.Println("")
+
+	fmt.Print("Overriden Environment Variables Check (ENDPOINT):\n----------------------------------\n")
+	for _, envVar := range endpointApp.Spec.Template.Spec.Containers[0].Env {
+		if strings.HasPrefix(envVar.Name, "CONFIG_JS_") {
+			fmt.Printf("    	✔ %-20s : %s\n", envVar.Name, envVar.Value)
+		}
+	}
```

Comment:
When there are no envs with `CONFIG_JS_` prefix we should print it out.

Reply:
sure, Thanks.